config/routes.rb

```diff
@@ -2,4 +2,5 @@
   get "/", to: 'welcome#index', as: :welcome
   # resources :welcome, :path => '/', only: :index, as: :welcome
 
+  get '/discover', to: 'search#top_forty'
```

Comment:
Should /discover route to an action called top_forty?

The way I imagined it:
get /discover, to: discover#show(or index)
on app/view/discover/index(or show).html.erb there would be a button for "top 40"
Clicking this button would take you to
get /movies/top_forty, to: search#top_forty

Reply:
Okay, so we get into the '/discover' URI, have it go to a Discover Controller action of #show
And the button exists on that show page.r
Once we click on that button, we use a movies/top_forty URI that leads to the Search Controller with a less restful path for the action top_forty.

If I got that right, I'll edit to make that possible.  It makes sense in my head.